vignettes/building-with-renv.Rmd

```diff
@@ -335,7 +335,7 @@ This will record the package in the lockfile so that the next time
 ## Updating the Cache
 
 If you want to update the packages in the package cache, you can use the
-function `fetch_updates()`. This requires an internet connection and will scan
+function `update_cache(prompt=TRUE)`. This requires an internet connection and will scan
```

Comment:
```suggestion
function `update_cache()`. This requires an internet connection and will scan
```

In general, when mentioning functions in text, I do not write the arguments they take. It's like mentioning someone by their full title and name instead of "my friend Dani".